src/CenterSystem.java

```diff
@@ -11,8 +11,11 @@
 
 public class CenterSystem extends UnicastRemoteObject implements CenterServer {
     private String centerName = "";
-    protected ConcurrentHashMap<Character, ArrayList<Records>> database = new ConcurrentHashMap<>();
+    protected  ConcurrentHashMap<Character,ArrayList<Records>> database = new ConcurrentHashMap<>();
```

Comment:
this must be changed to regular hashmap, now we have syncronized insert... but I'm not sure if we have syncronized edit